### ISSUE TITLE
add tooltips to all buttons

### ISSUE THREAD

**Issue by josephmcg** [Satellite-im/WarpGUI#484]
to be implemented after this merges
https://github.com/Satellite-im/Uplink/issues/481

may be better to see how i18n shakes out first

- add text to all instances of Button component for tooltip functionality. add `hide_text` prop to buttons as needed

**Comment by InfamousVague** [comment]
I am not sure we need it on every button, I couldn't find an example where that is done. But I do like the idea of adding it to buttons which are icon only. 

I think buttons which already include text should include an alt tag but that's fine, where as icon buttons should include the tooltip with smart positioning. (Or just specify the positioning in the props, optionally)

**Comment by josephmcg** [comment]
I initially leaned towards all buttons because I saw the text begin to overflow on mobile resulting in ellipsis. I saw that got fixed in a separate ticket though, so I agree that we can conditionally apply the title.

Unfortunately, alt isn't supported on svg. The best practice for accessible text on icon buttons is usually a div with a sr-only class. This applies styles to visually hide the text, but allow screen readers to still pick it up. The rules for this class have evolved over the years, so it can be dangerous copying something from an old stack overflow thread. The tailwind one is solid, so we can use that
https://tailwindcss.com/docs/screen-readers
